### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Prathapkarumanchi** [f-prime/Blunt#6]
Hi,

I have a questions, how this is different from Tailwind-CSS

**Comment by f-prime** [comment]
The two frameworks are similar in their goals. It depends what you need. If you need a fully well rounded CSS framework that gives you color helper classes and other _styling_ classes as well as layout classes, then Tailwind is the better bet. If you want a CSS framework that just provides layout classes then that's where Blunt comes in. Blunt's goal is to have no opinions on how your application should look,. This is left entirely up to you to create the CSS that does this. Button styles, borders, colors, etc. That is all left up to you. Tailwind gives you classes for this out of the box.

Though both frameworks are pretty heavy, Blunt is much lighter than tailwind which weighs in at over 1MB when minified https://cdnjs.cloudflare.com/ajax/libs/tailwindcss/1.4.6/tailwind.min.css. Whereas Blunt is about half of that https://cdn.jsdelivr.net/gh/f-prime/blunt/dist/blunt.min.css

Blunt has no intention of being put on the NPM registry and is intended to either be included directly in your HTML or downloaded manually and put into your static files. 

Blunt is written in SCSS whereas Tailwind uses Javascript to generate the CSS.

tl;dr; - They are similar but Blunt has a different philosophy than Tailwind. If you need something that has opinions on how your page should be styled use Tailwind, if you want to do the styling completely on your own use Blunt.
